sourcetracker/_sourcetracker.py

```diff
@@ -728,24 +734,19 @@ def gibbs(sources, sinks=None, alpha1=.001, alpha2=.1, beta=10, restarts=10,
     >>> burnin = 2
     >>> delay = 2
 
-    # Call without a cluster
     >>> mpm, mps, fas = gibbs(source_df, sink_df, alpha1, alpha2, beta,
                               restarts, draws_per_restart, burnin, delay,
-                              cluster=None, create_feature_tables=True)
+                              create_feature_tables=True)
 
-    # Start a cluster and call the function
-    >>> jobs = 4
-    >>> subprocess.Popen('ipcluster start -n %s --quiet' % jobs, shell=True)
-    >>> time.sleep(25)
-    >>> c = Client()
+    # Run the function on multiple processors
     >>> mpm, mps, fas = gibbs(source_df, sink_df, alpha1, alpha2, beta,
                               restarts, draws_per_restart, burnin, delay,
-                              cluster=c, create_feature_tables=True)
+                              jobs=4, create_feature_tables=True)
 
     # LOO prediction.
     >>> mpm, mps, fas = gibbs(source_df, sinks=None, alpha1, alpha2, beta,
```

Comment:
This examples doesn't work (your PR didn't break it I don't think). Needs to read 
```python
    # LOO prediction.
    >>> mpm, mps, fas = gibbs(source_df, sinks=None, alpha1=alpha1,
                              alpha2=alpha2, beta=beta, restarts=restarts,
                              draws_per_restart=draws_per_restart,
                              burnin=burnin, delay=delay, jobs=5,
                              create_feature_tables=True)
```